site/content/entry/yuzu-progress-report-jul-2023/index.md

```diff
@@ -0,0 +1,201 @@
+
+{{< imgs
+	"./sp3.png| She calls it Sasha (Splatoon 3)"
+  >}}
+
+# Input Changes
+
+[German77](https://github.com/german77) continued his reverse engineering of the Switch NFC service and {{< gh-hovercard "11096" "improved yuzu's code accuracy" >}}.
+He also fixed a bug which caused users' mouse to stop working after some time, by avoiding an accidental division by zero.
+
+Newcomer [SuperSamus's](https://github.com/SuperSamus) first contribution to yuzu was to {{< gh-hovercard "11050" "fix how button mapping was being handled within yuzu." >}}
+With these changes, it's now possible to share the same mappings between Nintendo and non-Nintendo controllers. Thank you!
+
+Note: This change **BREAKS** controller configurations for those using Nintendo controllers without the direct driver.
+
+# Android Changes
+
+[bunnei](https://github.com/bunnei/) implemented changes to {{< gh-hovercard "11017" "fix an issue where Turnip drivers would fail" >}} on certain Snapdragon devices, and added checks for broken Qualcomm 7xx drivers.
+This means any Snapdragon user should be able to run Turnip drivers now, links will be provided in the Hardware section later in this article.
+
+[t895](https://github.com/t895) {{< gh-hovercard "11067" "disabled yuzu from asking to save user data on uninstall," >}} which could cause issues when users attempted to use different APK versions.
+
+t895 also {{< gh-hovercard "11070" "added some help text messages" >}} to inform users why certain buttons were disabled when games were running.
+The most common case is Mali or Xclipse users trying to install custom drivers, a feature that is only available and possible to Adreno users, at least for now.
+
+{{< imgs
+	"./mali.png| Abstract art generator (Hyrule Warriors: Age of Calamity)"
+  >}}
+
+# Miscellaneous changes
+
+Apart from the ones highlighted above, there have also been several smaller changes:
+
+[FearlessTobi](https://github.com/FearlessTobi) added {{< gh-hovercard "11047" "detection for compressed and sparse NCAs within yuzu's filesystem." >}} Instead of failing silently, yuzu will now log this info and abort the parsing.
+
+toast added a {{< gh-hovercard "11042" "new Linux build script which added the libraries required to enable `Wayland` support" >}} on our official AppImage releases.
+
+Not ending there, toast {{< gh-hovercard "11186" "fixed a potential memory leak with the new timezone binary," >}} which happened when games tried to ask for timezone information too many times. 
+This was resolved by keeping track of the generated timezone binary instead of recreating it repeatedly.
+
+And toast also {{< gh-hovercard "11030" "bypassed a MSVC build crash on `Windows version 10 1809 LTSC` with the new timezone binary changes," >}} by temporarily disabling this for MSVC until there is a solution from Microsoft. 
+The actual reason for this crash on older Windows 10 builds is currently unknown.
```

Comment:
The reason is known. See https://github.com/microsoft/STL/issues/3853#issuecomment-1627630752 and related links